docs/user_guide/reductions.rst

```diff
@@ -0,0 +1,429 @@
+The Reductions Approach to Disparity Mitigation
+===============================================
+
+The "Reductions Approach" is named because it *reduces* the problem of training a
+classifier while satisfying a disparity constraint to that of training a classifier
+on weighted data.
+This technique was first described in [#1]_.
+
+
+Fair Classification
+-------------------
+
+Consider the standard setting for binary classification.
+In such a case, we wish to learn the classifer
+:math:`h \in \mathcal{H}` which minimises the classification
+error :math:`\mbox{err}(h) = \P[ h(X) \neq Y]`.
+Now, let us suppose that we can devise a constraint function
+:math:`\mathbf{C}(h)` which measures the violation of our
```

Comment:
We can define abbreviations in `conf.py`, e.g. `\H` for `\mathcal{H}`. If we do this for all reocurring ones that'll make it a lot more readable.